pkg/users/account-details.js

```diff
@@ -210,7 +168,7 @@ export function AccountDetails({ accounts, groups, shadow, current_user, user })
         cockpit.spawn(["/usr/bin/loginctl", "terminate-user", user],
                       { superuser: "try", err: "message" })
                 .then(() => {
-                    get_details(user).then(setDetails);
+                    get_expire(user).then(setExpiration);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17830-20221129-111828-0213c20d-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)